src/release/sim_reads.cpp

```diff
@@ -235,19 +235,20 @@ int64_t SimulateReads(Sequence &ref, int num_reads, FastaWriter &writer)
 //                if (seq.IsValid())
 //                    break;
             }
+			if(seq.size()>read_length){
```

Comment:
@Malfoy Thanks for the fix. Simply skipping invalid sequence will result in less output. I think it is better to do the following
```
if (seq.size() < read_lengt) {
  continue;
}
```